apps/demo-angular/src/plugin-demos/wcg-text-encoding.component.ts

```diff
@@ -0,0 +1,17 @@
+import { Component, NgZone } from '@angular/core';
+import { DemoSharedWcgTextEncoding } from '@demo/shared';
+import {} from '@nativescript/wcg-text-encoding';
```

Comment:
Is this an import for effects, or unneeded?
```suggestion
import '@nativescript/wcg-text-encoding';
```

Reply:
This is due to the workspace add package cmd